### PR TITLE
Added the Contributor Covenant Code of Conduct

### LINKED ISSUE #57: Add a code of conduct
I plan to place the project under the [Contributor Covenant](https://www.contributor-covenant.org/). Anyone against it ?

### PR DESCRIPTION
**Pull request by pmourlanne**
Closes #57

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+an individual is representing the project or its community in public spaces.
+Examples of representing a project or community include using an official
+project e-mail address, posting via an official social media account, or acting
+as an appointed representative at an online or offline event. Representation of
+a project may be further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at placeholder@ultimatesoftware.com. All
```
Comment (pmourlanne) [inline CODE_OF_CONDUCT.md]: We need to decide on an email address :o

Comment (ewjoachim) [reply CODE_OF_CONDUCT.md]: I'm ok with my own address and maybe someone else's too so that if I screw up, they can speak to someone else. You ? @mgu ?

Comment (pmourlanne) [reply CODE_OF_CONDUCT.md]: By your own address you mean your `@ultimatesoftware` address?
I'm OK with adding my own address, but it won't be the `@ultimatesoftware` one though, doesn't seem future-proof :)

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+Further details of specific enforcement policies may be posted separately.
+
+Project maintainers who do not follow or enforce the Code of Conduct in good
+faith may face temporary or permanent repercussions as determined by other
+members of the project's leadership.
+
+## Contacts
+
+- Joachim Jablon - @ewjoachim (first name arobase last name dot fr)
+- Pierre Mourlanne - @pmourlanne (...?)
```
Comment (ewjoachim) [inline CODE_OF_CONDUCT.md]: @pmourlanne new proposal, how do you like it ?

Comment (pmourlanne) [reply CODE_OF_CONDUCT.md]: I added my email address :+1:

**Comment by ewjoachim** [comment]
Let's merge as-is, we can always iterate, but an existing CoC is better than nothing.

**Comment by ewjoachim** [comment]
Haha, faster than me :D

**Pull request merged**